apps/root/src/frame/components/earn-gain-access-modal/index.tsx

```diff
@@ -0,0 +1,254 @@
+`;
+
+const StyledHeaderContent = styled(ContainerBox).attrs({ gap: 3, alignItems: 'center' })`
+  ${({ theme: { spacing } }) => `
+    padding: ${spacing(5)} ${spacing(12)};
+  `}
+`;
+
+const StlyedRainCoins = styled.div`
+  background: url('https://ipfs.io/ipfs/QmSeVzhz72iHZGvq9pinvKeGScyQR85JqyBeahxVVs9Tuq') 50% / contain no-repeat;
+  width: 250.26px;
+  height: 227.044px;
+  position: absolute;
+  right: 7.996px;
+  bottom: -72.098px;
+`;
+
+const StyledFeatureAssetFirst = styled.div`
+  position: absolute;
+  background: url('https://ipfs.io/ipfs/QmRknhCx7rWTCABFqYRdPm7iicaAJMWazGGDFXhygXnVTa') 50% / contain no-repeat;
+  width: 198px;
+  height: 197px;
+  transform: rotate(15.309deg);
+  left: -68px;
+  bottom: -80px;
+`;
+
+const StyledFeatureAssetSecond = styled.div`
+  position: absolute;
+  background: url('https://ipfs.io/ipfs/QmTFLrcx2aTWeJKHCwCDU2yqLf44DcYZbrVsjrNP46vsR6') 50% / contain no-repeat;
+  width: 289px;
+  height: 289px;
+  transform: rotate(-40.909deg);
+  left: -166px;
+  bottom: -171px;
+`;
+
+const StyledFeatureAssetThird = styled.div`
+  position: absolute;
+  background: url('https://ipfs.io/ipfs/QmYMtqhdUet6EMN8WX1GF667AU81Q7DYVvGDE8f9By23Uw') 50% / contain no-repeat;
+  width: 242px;
+  height: 242px;
+  transform: rotate(71.567deg);
+  left: -61px;
+  bottom: -130px;
+`;
+
+interface FeatureCardProps {
+  title: string;
+  description: string;
+  mode: 'light' | 'dark';
+  FeatureAsset: React.ComponentType;
+}
+
+const getGradientColor = (mode: 'light' | 'dark') => {
+  return mode === 'light'
+    ? 'linear-gradient(180deg, #EBE4F5 0%, #BAB2FF 100%)'
+    : 'linear-gradient(180deg, #2E2040 0%, #181122 100%)';
+};
+
+const FeatureCard = styled(({ title, description, FeatureAsset, ...props }: FeatureCardProps) => (
+  <ContainerBox {...props} gap={1} flexDirection="column">
+    <Typography variant="bodySmallSemibold" color={({ palette }) => colors[palette.mode].typography.typo1}>
+      {title}
+    </Typography>
+    <Typography variant="bodySmallRegular" color={({ palette }) => colors[palette.mode].typography.typo2}>
+      {description}
+    </Typography>
+    <FeatureAsset />
+  </ContainerBox>
+))`
+  ${({ theme: { palette, spacing } }) => `
+    padding: ${spacing(6)} ${spacing(6)} ${spacing(19)};
+    border-radius: ${spacing(4)};
+    background: ${getGradientColor(palette.mode)};
+    height: 100%;
+    position: relative;
+    overflow: hidden;
+  `}
+`;
+
+const EarnGainAccessModal = ({ isOpen, onClose }: EarnGainAccessModalProps) => {
+  const intl = useIntl();
+  const activeWallet = useActiveWallet();
+  const wallets = useWallets();
+  const openConnectModal = useOpenConnectModal();
+  const pushToHistory = usePushToHistory();
+  const themeMode = useThemeMode();
+
+  const handleAccessNow = () => {
+    pushToHistory(`/${EARN_ACCESS_NOW_ROUTE.key}`);
+    onClose();
+  };
+
+  const actions = React.useMemo(
+    () =>
+      [
+        ...(!activeWallet?.address && !wallets.length
```

Comment:
from what we've seend with the designs, the user would not need to connect here, since we can handle it on the landing page